hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/ContainerManagerImpl.java

```diff
@@ -311,6 +318,10 @@ protected void createAMRMProxyService(Configuration conf) {
     }
   }
 
+  protected ContainerScheduler createContainerScheduler(Context cntxt) {
```

Comment:
Is this method exposed for testing? If yes, should it be marked @VisibleForTesting?